package/yast2.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Feb 24 15:02:15 CET 2020 - schubi@suse.de
+
+- Reinitializing Augeas parser again before writing data to
```

Comment:
It seems that the fix is now rather "create an own Augeas instance" than "reinitializing"...